src/tester/get-affected-packages.ts

```diff
@@ -124,10 +119,10 @@ async function gitDiff(log: Logger, options: Options): Promise<string[]> {
 
 	// `git diff foo...bar` gets all changes from X to `bar` where X is the common ancestor of `foo` and `bar`.
 	// Source: https://git-scm.com/docs/git-diff
-	let diff = (await run(`git diff ${sourceBranch}...HEAD --name-only`)).trim();

```

Comment:
Don't need the `...HEAD`; this enables running the tester on non-committed changes.